samples/triangle/main.c

```diff
@@ -52,6 +52,10 @@ void main(void)
     int       start, last, now;
     int       fps, frames, frames_total;
 
+    XVideoSetMode(640, 480, 16, REFRESH_DEFAULT);
+
+    pb_set_color_format(NV097_SET_SURFACE_FORMAT_COLOR_LE_R5G6B5, false);
+
```

Comment:
This can be removed. It's only for testing.
But we might as well keep it to demo how to use this?

Alternative combinations:
```c
    XVideoSetMode(640, 480, 16, REFRESH_DEFAULT);
    pb_set_color_format(NV097_SET_SURFACE_FORMAT_COLOR_LE_X1R5G5B5_Z1R5G5B5, false);
```
```c
    XVideoSetMode(640, 480, 16, REFRESH_DEFAULT);
    pb_set_color_format(NV097_SET_SURFACE_FORMAT_COLOR_LE_X1R5G5B5_O1R5G5B5, false);
```
```c
    XVideoSetMode(640, 480, 32, REFRESH_DEFAULT);
    pb_set_color_format(NV097_SET_SURFACE_FORMAT_COLOR_LE_A8R8G8B8, false);
```

*(Last one is default if it's omitted)*

Reply:
Waiting for removal

Reply:
Removed.